README.md

```diff
@@ -34,6 +34,7 @@ Following is a list of Greengrass components.
 * [TES Routing to Container](https://github.com/awslabs/aws-greengrass-labs-tes-router): This component configures nftables or iptables on a Greengrass device to be able to use [Token Exchange Service](https://docs.aws.amazon.com/greengrass/v2/developerguide/token-exchange-service-component.html) component with containers.
 * [WebRTC](https://github.com/awslabs/aws-greengrass-labs-webrtc): This component ingests audio and video streams from RTSP cameras connected to the Greengrass core device. And then the component turns the audio and video streams into peer-to-peer communication or relay through Amazon Kinesis Video Streams.
 * [GStreamer for LookoutForVision](https://github.com/awslabs/aws-greengrass-labs-lookoutvision-gstreamer): This component provides a GStreamer plugin that enables you to perform [Amazon Lookout For Vision](https://aws.amazon.com/lookout-for-vision/) anomaly detection in your custom GStreamer pipelines.
+* [Secrets Manager client](https://github.com/awslabs/aws-greengrass-labs-secretsmanagerclient): This component provides a CLI tool that can be used by other components needing to retrieve secrets from the SecretManager component in a recipe lifecycle script.
```

Comment:
Could you add commands using gdk in the readme of the Secrets manager repo?
Also, since the issue with gradle command is fixed in 1.1.0, could you change the version here https://github.com/awslabs/aws-greengrass-labs-secretsmanagerclient/blob/main/gdk-config.json#L16 to 1.1.0?

Reply:
Instructions to build the component are in BUILD.md and mention gdk already.
I have updated the version in the gdk-config.json

Reply:
Thanks Massi